components/base/wheeled/sensorbase.go

```diff
@@ -43,8 +43,9 @@ type sensorBase struct {
 
 	opMgr operation.SingleOperationManager
 
-	allSensors  []movementsensor.MovementSensor
-	orientation movementsensor.MovementSensor
+	allSensors       []movementsensor.MovementSensor
+	orientation      movementsensor.MovementSensor
+	velocitiesSensor movementsensor.MovementSensor
```

Comment:
no - just like adding an encoder to a motor wraps the motor in feedback logic, this will wrap wheeledbase in feedback logic. You can still make a wheeled base, notice there's not creator function in this file.